tgui/packages/tgui/interfaces/Quickload_SOM/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useBackend, useLocalState } from "../../backend";
+import { Stack, Button, Section, Box, LabeledList, Modal, Tabs, Flex } from "../../components";
+import { Window } from "../../layouts";
+import { LoadoutListData, LoadoutTabData, LoadoutManagerData, LoadoutItemData } from './Types';
+
+const LoadoutItem = (props : LoadoutItemData, context) => {
```

Comment:
What you did for the UIs makes no sense to me
Why didnt you just send the vendor faction to the UI and then have the UI use dicts to determine what styles/descs/names etc to use

Reply:
The ui is loadout vendor code that I surgically mutilated with a sharp rock, I don't actually have a good grasp on how it works. If you could run me through it some time though, I'd be happy to change it.